Dockerfile.Nextcloud

```diff
@@ -0,0 +1,11 @@
+ARG NEXTCLOUD_VERSION=22.2.0
```

Comment:
Nextcloud v22 is a very old version (approx 3 years old), probably not the best default?